docs/development/releasing.rst

```diff
@@ -115,7 +115,12 @@ packages that use the full bugfix/maintenance branch approach.)
 
    Don't forget to remove any non-committed files with::
 
-      $  git clean -dfx
+      $ git clean -dfx
+
+#. Make sure the source distribution doesn't inherit limited permissions
+   following you default umask::
+
+     $ umask 0022
```

Comment:
@pllim - are you really doing a release from windows? I'm not sure how it deals with permissions in general, but probably the easiest is to run an sdist and check the result on a different machine.

However my plan is to deal with it in a pythonic way in helpers, and that may be insensitive to the os.

@MSeifert04 - this is totally independent of git, and relates to how ``setup.py`` works with the packaging.